### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by yichaoS**
- fixes based on feedback
- add maven plugin for installing npm and node
- removed ghost "specRunner.html"

### CODE REVIEW

File: portal/src/main/webapp/js/src/cbio-stat.js [hunk 1]
```diff
@@ -1,14 +1,12 @@
-if (cbio === undefined) {
```
Comment (zhx828) [inline portal/src/main/webapp/js/src/cbio-stat.js]: We shouldn't remove initialization here. I notice you declare the cbio in cbio-stat.spec.js but we will not include cbio-stat.spec.js in the final js build file, right?

File: portal/src/main/webapp/js/SpecRunner.html [hunk 2]
```diff
@@ -1,99 +0,0 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
```
Comment (zhx828) [inline portal/src/main/webapp/js/SpecRunner.html]: Is it no longer needed?

Comment (yichaoS) [inline portal/src/main/webapp/js/SpecRunner.html]: No. This is an independent page showing some old test results based on jasmine but even those tests are not working anymore. Here is the screenshot. 
image

File: portal/src/main/webapp/js/src/cbio-stat.js (hunk 1 above)
Comment (yichaoS) [inline portal/src/main/webapp/js/src/cbio-stat.js]: No it's not. Added it back.

**Pull request merged**